src/bci_build/package/basecontainers.py

```diff
@@ -43,7 +43,7 @@ def _get_eula_package_names(os_version: OsVersion) -> tuple[str, ...]:
         package_name="micro-image",
         logo_url="https://opensource.suse.com/bci/SLE_BCI_logomark_green.svg",
         is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
-        pretty_name=f"{os_version.pretty_os_version_no_dash} Micro",
+        pretty_name=f"{os_version.pretty_os_version_no_dash} Micro".strip(),
```

Comment:
Maybe add the `strip()` to `__post_init__` instead? Imho there's no danger in stripping trailing and leading whitespace.

Reply:
ah, good point, I was searching for a place where it is less ugly..